src/style.css

```diff
@@ -1,3 +1,70 @@
+html {
+  font-family: Roboto, sans-serif;
+
+  --main-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: #fff; /* change later */
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+  padding-right: 50px;
+}
+
+.logo {
+  margin-left: 50px;
+}
+
+.logo__img {
+  display: block;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  position: relative;
+}
+
+.nav__link {
+  display: inline-block;
+  height: 60px;
+  line-height: 60px;
+  text-decoration: none;
+  color: #000;
+  text-transform: uppercase;
+  font-size: 12px;
+  margin: 0 0 0 20px;
+}
+
+.nav__link:hover {
+  color: var(--main-color);
+}
+
+.nav__list > :first-child .nav__link {
+  margin-left: 0;
+}
+
+.is-active {
+  position: relative;
+  color: var(--main-color);
+}
+
+.is-active::after {
+  position: absolute;
+  content: "";
+  background-color: var(--main-color);
+  border-radius: 8px;
+  width: 100%;
+  height: 4px;
+  bottom: 0;
+  left: 0;
```

Comment:
now you should remove `left: 0` and `padding: 0 -10px` cause you already have `display:block`